server/src/test/java/org/opensearch/action/support/master/TransportMasterNodeActionTests.java

```diff
@@ -419,11 +419,11 @@ public void testDelegateToFailingMaster() throws ExecutionException, Interrupted
         boolean failsWithConnectTransportException = randomBoolean();
         boolean rejoinSameMaster = failsWithConnectTransportException && randomBoolean();
```

Comment:
`rejoinSameMaster ` -> `rejoinSameClusterManager`

Reply:
Changed this variable name. 👍